deployment_botpress/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "1"
+services:
+  botpress:
+    image: botpress/server
+    expose:
+      - 3000
+    ports:
+      - "3000:3000"
```

Comment:
Do we need this? Isn't it all through nginx and doesn't need to be exposed outside of it?